@xen-orchestra/xapi/src/vm.js

```diff
@@ -402,6 +403,27 @@ module.exports = class Vm {
     }
   }
 
+  @cancelable
```

Comment:
Let's not use `cancelable` anymore and add an optional `cancelToken` option instead.

IMHO it's easier to use.